src/invisible/how-tos/advanced-analysis/intrinsics/README.ipynb

```diff
@@ -201,6 +201,7 @@
     "To undistort a full recording we simply need to apply the undistortion to every video frame and all gaze data. This requires a good bit of additional code to handle the video correctly, but works just the same.\n",
     "\n",
     "You can download the recording used in this example [here](https://drive.google.com/file/d/1CGVgP6GWbEESoJUMB2YiD_gNN4JicK75/view?usp=sharing).\n",
+    "Extract the downloaded content into the `data` folder.\n",
```

Comment:
What is "the data folder" though? Users do not have a location that contains a data folder. They need to customize their paths anyway, so I am not sure if this is helpful.